provider/rfc2136/rfc2136.go

```diff
@@ -389,7 +389,6 @@ func (r rfc2136Provider) SendMessage(msg *dns.Msg) error {
 	log.Debugf("SendMessage")
 
 	c := new(dns.Client)
-	c.SingleInflight = true
```

Comment:
I'm presuming we call `SendMessage()` in serial, so we don't have to implement this functionality.

Reply:
Yes, that's also my understanding.